src/docs/economy.mdx

```diff
@@ -21,10 +21,10 @@ The item-based currency allow players to dig it up straight from the ground. Gol
 All town protections are removed when a town falls. Find abandoned towns and loot the riches. Stealing is allowed.
 
 ### Voting
-Run the command `/vote` and click the provided links. You can vote right now by visiting the voting sites [here](https://minecraftservers.org/server/383495) and [here](https://servers-minecraft.net/server-earthmc.1042/vote). You will receive a gold crate with a random award of up to 64 gold ingots after successfully voting for the server. Keep in mind that you are able to vote without being online. Your award will be given to you next time you sign in and can be claimed with `/cr claim`. The most likely award is 4 gold per vote link. It's also possible to win tridents and mending enchantment books from voting although they are very rare.
+Run the command `/vote` and click the two links provided to you. You will recieve a gold crate for each site you successfully submit a vote to. Currently, you can recieve up to two gold crates per 24-hour day cycle. You can recieve up to 64 gold or a special item reward on each gold crate you earn. Keep in mind that you are able to vote without being online. If you decide to vote while you're not on the server, your award will be given to you next time you sign in and run the command `/cr claim`.
```

Comment:
```suggestion
Run the command `/vote` and click the two links provided to you. You will receive a gold crate for each site you successfully submit a vote to. Currently, you can receive up to two gold crates per 24-hour day cycle. You can receive up to 64 gold or a special item reward on each gold crate you earn. Keep in mind that you are able to vote without being online but that you will not receive any reward.
```

Reply:
Last sentence: "Remember that you can vote even if you're not online, but you won't receive any rewards."